homeassistant/components/zha/core/typing.py

```diff
@@ -10,33 +10,8 @@
 
 # pylint: disable=invalid-name
 CALLABLE_T = TypeVar("CALLABLE_T", bound=Callable)
-ChannelType = "ZigbeeChannel"
-ChannelsType = "Channels"
-ChannelPoolType = "ChannelPool"
-ClientChannelType = "ClientChannel"
-ZDOChannelType = "ZDOChannel"
-ZhaDeviceType = "ZHADevice"
-ZhaEntityType = "ZHAEntity"
-ZhaGatewayType = "ZHAGateway"
-ZhaGroupType = "ZHAGroupType"
```

Comment:
These are the types that are causing issues for `mypy`.
I have removed them, and corrected the other modules accordingly.